inference-engine/src/gna_plugin/runtime/pwl.h

```diff
@@ -21,7 +21,8 @@
 #define IDENTITY_NUM_SEGMENTS 3
 #define IDENTITY_DOMAIN 10.0f
 #define PWL_MAX_ERR_PERCENT 1.0f
-#define PWL_MAX_ITERATIONS 2000
+#define PWL_MAX_ITERATIONS_DEFAULT 2000
+#define PWL_MAX_ITERATIONS_LOG 5000
```

Comment:
I'm a bit confused by this change, why we are increasing it for LOG only? I know that @dmitriikhurtin  is working on a generic improvement for this logic. Don't we need to wait for his commit?

Reply:
It was found in customer's networks that 2000 iterations are not enough for Log activation to get the appropriate accuracy for the network. As I understand the generic improvement supposes removal of limitation for iterations number at all. This approach should be tested separately on the real networks to check accuracy improvement and performance degradation. So I think for now we can start with increasing of this parameter for Log only.